source/guide/3.2/ui-blueprint-upload.md

```diff
@@ -36,14 +36,15 @@ The upload blueprint button can be found in the "Blueprints" section in the UI:
 ![The blueprint upload button](/guide/images/ui/ui_upload_blueprint_button.png)
```

Comment:
Also, the sentence: Cloudify's UI allows the user to upload a packaged blueprint TAR file... is written twice. Once in the beginning and once under #uploading a blueprint.